Example/NYTPhotoViewer.xcodeproj/xcshareddata/xcschemes/NYTPhotoViewer.xcscheme

```diff
@@ -23,10 +23,11 @@
       </BuildActionEntries>
    </BuildAction>
    <TestAction
+      buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
       shouldUseLaunchSchemeArgsEnv = "YES"
-      buildConfiguration = "Debug">
+      codeCoverageEnabled = "YES">
```

Comment:
I haven't figured out how to get the code coverage measurement to apply to the Pods target, just the example application :/